data/coops/tig.yaml

```diff
@@ -0,0 +1,12 @@
+name: TIG Bicimensajería
+url: 'https://www.tigcdmx.com/'
+facebook_url: 'https://www.facebook.com/tig.mx'
+coopcycle_url: 'https://tig.coopcycle.org'
+mail: info@tig.mx
+city: Ciudad de México
```

Comment:
The problem here is that the `city` property is used in the app to build the list of cities. 
There is already TWC which has [the same value](https://github.com/coopcycle/coopcycle-website/blob/b291fa030bd026c8c9988b18f43c2ad82fbca15c/data/coops/twc.yaml#L4), so "Ciudad de México" would appear twice. 

At the moment, the solution is to write it like "city - name", like for Naofood & Les Coursiers Nantais. 

https://github.com/coopcycle/coopcycle-website/blob/b291fa030bd026c8c9988b18f43c2ad82fbca15c/data/coops/naofood.yml#L10

https://github.com/coopcycle/coopcycle-website/blob/b291fa030bd026c8c9988b18f43c2ad82fbca15c/data/coops/les_coursiers_nantais.yml#L7